src/test/api_compatibility.ts

```diff
@@ -3,8 +3,9 @@ import { assert, expect } from 'chai';
 import { BinaryCarrier, FORMAT_BINARY, FORMAT_TEXT_MAP, Reference, REFERENCE_CHILD_OF, Span, Tracer } from '../index';
 
 export interface ApiCompatibilityChecksOptions {
-    /** a boolean that controls whether or not to verify baggage values */
-    checkBaggageValues?: boolean;
+    /** a boolean that controls whether or not to verify certain API functionality */
+    skipCheckBaggageValues?: boolean;
+    skipCheckInjectBehaviour?: boolean;
```

Comment:
I am being a bit pedantic about the naming, because it constitutes a public API.
  * skipCheckBaggageValues used to control only reading the baggage, now it's a larger scope.
  * skipCheckInjectBehaviour controls both inject & extracy
  * I would prefer to have a uniform naming scheme for these flags, e.g. `skip{Something}Checks`, resulting in
    * skipBaggageChecks
    * skipInjectExtractChecks